src/main/java/net/openhft/chronicle/queue/impl/single/SingleChronicleQueueBuilder.java

```diff
@@ -269,7 +269,7 @@ static SingleChronicleQueueStore createStore(@NotNull RollingChronicleQueue queu
         return wireStore;
     }
 
-    static boolean isQueueReplicationAvailable() {
+    public static boolean isQueueReplicationAvailable() {
```

Comment:
Should we make an alias for this `areEnterpriseFeaturesAvailable` and deprecate this one, if we're expanding its use beyond replication (just looks a bit WTF when you're asking that in the context of the pretoucher)

Reply:
In fact given it wasn't public previously we could probably get away with just renaming it.

Reply:
yep - agree